src/lwc/rd2EntryForm/rd2EntryForm.css

```diff
@@ -11,4 +11,13 @@
     width: 100%;
     text-align: center;
     top: calc(50% + 3em);
+}
+
+.slds-modal__content {
```

Comment:
It seems the width of the modal is smaller than previously, this when the Recurring Period is Advanced, the Installment Period is not aligned with other fields.
@ravensisco , please add into the test charter a test for a user with non-English locale and see how fields are rendered. Also, please ensure the form is rendered as expected in different browsers.

<img width="621" alt="Screen Shot 2020-11-04 at 10 15 17 AM" src="https://user-images.githubusercontent.com/28743286/98153454-10badf80-1e88-11eb-8b85-88773e6c74f3.png">

Reply:
add modal-small to increase the width